torch/_subclasses/fake_impls.py

```diff
@@ -396,6 +396,11 @@ def local_scalar_dense(fake_mode, func, arg):
     return r
 
 
+@register_op_impl(torch.ops.aten.nonzero_numpy.default)
+def nonzero_numpy(fake_mode, func, arg):
+    return torch.ops.aten.nonzero.default(arg).unbind(1)
+
+
```

Comment:
@tugsbayasgalan helped me with adding this since nonzero_numpy was appearing in one of the `export_for_training` tests, @ezyang would this be ok?

Reply:
I don't even know what this does but assuming you did it right suuuuure